File: 2023 czerwiec/INF.04-02-23.06-SG/Aplikacja Konsolowa/C++/main.cpp
#include <iostream>
#include <random>

using namespace std;

void sorter(int numbers[])
{
    int temp;

    for (int i = 0; i < 99; i++)
    {
        for (int j = 0; j < 99 - i; j++)
        {
            if (numbers[j] > numbers[j + 1])
            {
                temp = numbers[j];
                numbers[j] = numbers[j + 1];
                numbers[j + 1] = temp;
            }
        }
    }
}

int main()
{
    random_device rd;
    mt19937 gen(rd());
    uniform_int_distribution<> distr(1, 1000);

    int numbers[100];

    cout << "Tablica przed sortowaniem: " << endl; 

    for (int i = 0; i < 100; i++)
    {
        numbers[i] = distr(gen);
        cout << numbers[i] << ", ";
    }

    sorter(numbers);

    cout << endl << "Posortowana tablica: " << endl;

    for (int i = 0; i < 100; i++)
    {
        cout << numbers[i] << ", ";
    }

    return 0;
}

File: 2023 czerwiec/INF.04-02-23.06-SG/Aplikacja Konsolowa/Python/main.py
import random

def sorter(numbers):
    for i in range(99):
        for j in range(99 - i):
            if numbers[j] > numbers[j + 1]:
                numbers[j], numbers[j + 1] = numbers[j + 1], numbers[j]

numbers = {}

print("Tablica przed sortowaniem: ")

for i in range(100):
    numbers[i] = random.randrange(1, 1000)
    print(numbers[i], ", ", end="", sep="")
    
sorter(numbers)

print("\nPosortowana tablica: ")

for i in range(100):
    print(numbers[i], ", ", end="", sep="")